build/lib/i18n.ts

```diff
@@ -85,6 +299,26 @@ const iso639_3_to_2: Map<string> = {
 	'trk': 'tr'
 };
 
+const iso639_2_to_3: Map<string> = {
```

Comment:
We might want to consider using 2 letter id in the future. The reason why we have this mapping is that MS internally uses 3. If all is public we could switch over to 2.